lib/elastic_apm/transport/serializers/metadata_serializer.rb

```diff
@@ -10,7 +10,8 @@ def build(metadata)
             metadata: {
               service: build_service(metadata.service),
               process: build_process(metadata.process),
-              system: build_system(metadata.system)
+              system: build_system(metadata.system),
```

Comment:
Sidenote: Do you prefer dangling commas? I see a lot of smart people preferring them to minimise git churn like this one. I see both sides, I guess. Curious if you've ever contemplated the issue?

Reply:
I'm fine with it, if you'd like. We can also add it to rubocop to enforce it.

Reply:
I don't think we should change anything now. Just curious for your thoughts 😊